test/extended/authentication/sync-schema2-01.yaml

```diff
@@ -0,0 +1,12 @@
+kind: LDAPSyncConfig
+apiVersion: v1
+url: ldap://LDAP_SERVICE_IP:389
+insecure: true
+activeDirectory:
+    usersQuery:
+        baseDN: "ou=people,ou=ad,dc=example,dc=com"
+        scope: sub
+        derefAliases: never
+        filter: (objectclass=organizationalPerson)
+    groupMembershipAttributes: [ mail ]
```

Comment:
This will change once https://github.com/openshift/origin/pull/4923#issuecomment-145654119 is addressed.